LEAF_Request_Portal/sources/EmailTemplate.php

```diff
@@ -168,16 +180,15 @@ public function getHistory($filterByName)
 
     public function setEmailTemplate($template)
     {
-        if (!$this->login->checkGroup(1))
-        {
+        error_log(print_r('setEmailTemplate - Template Name: '. $template, true));

```

Comment:
Why are you dumping to the error log here?

Reply:
Has been removed